ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -130,14 +136,44 @@ private T GetSetting<T>(string key, T defaultValue)
                     return (T)data;
                 }
             }
+#else
+            if (typeof(T) == typeof(bool))
+            {
+                try
+                {
+                    Boolean result = Boolean.TryParse(ConfigurationManager.AppSettings[key], out result);
+                    return (T) (object) result;
+                }
+                catch (ConfigurationErrorsException)
+                {
+
+                }
```

Comment:
Probably, it was in the example I stole this from and they were doing a Console.write which didn't seem useful but I couldn't think of anything better to use so I figured I'd leave it blank to see if you had any suggestions.